sos/report/plugins/openstack_nova.py

```diff
@@ -215,15 +216,46 @@ def setup(self):
             "/etc/polkit-1/localauthority/50-local.d/50-nova.pkla",
             "/etc/sudoers.d/nova",
             "/etc/security/limits.d/91-nova.conf",
-            "/etc/sysconfig/openstack-nova-novncproxy"
+            "/etc/sysconfig/openstack-nova-novncproxy",
+            "/var/lib/openstack/config/nova",
+            "/var/lib/openstack/containers/nova*.json"
         ])
+
         if self.get_option("all_logs"):
             self.add_copy_spec([
                 "/var/log/httpd/placement*",
+                "/var/log/containers/nova/*"
             ])
         else:
             self.add_copy_spec([
                 "/var/log/httpd/placement*.log",
+                "/var/log/containers/nova/*.log"
             ])
 
+        self.add_forbidden_path([
+            "/var/lib/openstack/config/nova/ssh-privatekey"
+        ])
+
+        def postproc(self):
```

Comment:
Ah, my bad here. Overlooked this, but the code scanning is flagging this because `postproc` is overindented and is part of `setup()` as written, where instead it should be just another method with the class.

Reply:
Ack!